x-pack/plugins/secops/public/components/page/hosts/host_summary/index.tsx

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiDescriptionList,
+  EuiDescriptionListTitle,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiPanel,
+  EuiTitle,
+  EuiToolTip,
+} from '@elastic/eui';
+import { FormattedRelative } from '@kbn/i18n/react';
+import { get, getOr, isArray } from 'lodash/fp';
+import React from 'react';
+import { pure } from 'recompose';
+import { HostItem, HostsEdges } from 'x-pack/plugins/secops/server/graphql/types';
+import { DragEffects, DraggableWrapper } from '../../../drag_and_drop/draggable_wrapper';
+import { escapeDataProviderId } from '../../../drag_and_drop/helpers';
+import { getEmptyValue } from '../../../empty_value';
+import { Provider } from '../../../timeline/data_providers/provider';
+import * as i18n from './translations';
+
+interface OwnProps {
+  data: HostsEdges[];
+  loading: boolean;
+  startDate: number;
+  endDate: number;
+}
+
+type HostSummaryProps = OwnProps;
+
+export const HostSummary = pure<HostSummaryProps>(({ data, startDate, endDate, loading }) => (
+  <EuiFlexItem style={{ maxWidth: 750 }}>
+    <EuiPanel>
+      <EuiTitle size="s">
+        <h3>{i18n.SUMMARY}</h3>
+      </EuiTitle>
+
+      <EuiHorizontalRule margin="xs" />
+      {getEuiDescriptionList(getOr(null, 'node', data[0]), startDate, endDate)}
+    </EuiPanel>
+  </EuiFlexItem>
+));
+
+const fieldTitleMapping: Readonly<Record<string, string>> = {
+  'host.name': i18n.NAME,
+  firstSeen: i18n.LAST_BEAT,
+  'host.id': i18n.ID,
+  'host.ip': i18n.IP_ADDRESS,
+  'host.mac': i18n.MAC_ADDRESS,
+  'host.type': i18n.TYPE,
+  'host.os.platform': i18n.PLATFORM,
+  'host.os.name': i18n.OS_NAME,
+  'host.os.family': i18n.FAMILY,
+  'host.os.version': i18n.VERSION,
+  'host.architecture': i18n.ARCHITECTURE,
+};
+
+const dateFields: string[] = ['firstSeen'];
```

Comment:
I'd name this `relativeFields` to match your function below so it's clearer.

Reply:
I had named them the same, but tslint gives you the ole `[tslint] Shadowed name: 'relativeFields' [no-shadowed-variable]`, so I ended up changing them.

Reply:
ok, that's understandable.